cmd/tools/diskscollector/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This command finds and removes orphaned disks created by the `kyma-gke-integration` job in a Google Cloud Platform (GCP) project.
+This command finds and removes orphaned disks created by the `kyma-gke-integration` job in a Google Cloud (GCP) project.
```

Comment:
again, I'm not sure GCP is still a correct abbreviation

Reply:
correct, it's not, good catch
Google Cloud Platform is now called Google Cloud: https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud

GCP could be abbreviation for Google Cloud services.
I think Google Cloud (GCP) it's not correct, it should be rather Google Cloud (GC) project if abbreviation is needed.